en/docs/learn/examples/jms-examples/dual-channel-http-to-jms.md

```diff
@@ -34,16 +33,15 @@ Shown below is the `SMSSenderProxy` proxy service.
              <property name="transport.jms.ContentTypeProperty"
                        value="Content-Type"
                        scope="axis2"/>
-          </inSequence>
-          <outSequence>
              <property name="TRANSPORT_HEADERS" scope="axis2" action="remove"/>
-             <send/>
-          </outSequence>
-          <endpoint>
-             <address uri="jms:/SMSStore?transport.jms.ConnectionFactoryJNDIName=QueueConnectionFactory&amp;java.naming.factory.initial=org.apache.activemq.jndi.ActiveMQInitialContextFactory&amp;java.naming.provider.url=tcp://localhost:61616&amp;transport.jms.DestinationType=queue&amp;transport.jms.ReplyDestination=SMSReceiveNotificationStore"/>
-          </endpoint>
+             <call>
+                <endpoint>
+                  <address uri="jms:/SMSStore?transport.jms.ConnectionFactoryJNDIName=QueueConnectionFactory&amp;java.naming.factory.initial=org.apache.activemq.jndi.ActiveMQInitialContextFactory&amp;java.naming.provider.url=tcp://localhost:61616&amp;transport.jms.DestinationType=queue&amp;transport.jms.ReplyDestination=SMSReceiveNotificationStore"/>
+                </endpoint>
+			    </call>
```

Comment:
Can you check the formatting
![Uploading Screenshot 2024-08-01 at 08.21.32.png…]()